packages/documentation-site/patternfly-docs/content/developer-resources/accessibility-development.md

```diff
@@ -1,116 +1,125 @@
+To render icons as something that assistive devices can ignore, pass `aria-hidden='true'` to all icons, either to the icon element or a parent element of the icon. 
 
-When accessibility is concerned, there are two main categories of icons:
-- **Decorative icons** are icons that can be removed without affecting the page's information.
-- **Semantic icons** are icons that provide information that isn't present on the page. For example, semantic icons can represent status, indicate an alert message type, or replace text as button labels.
+#### **Decorative and semantic icons**
 
-The meaning of a semantic icon must be provided in alternative ways to the user. Additionally, follow these guidelines for semantic icons:
+There are two main categories of accessible icons:
+1. **Decorative icons**, which can be removed without affecting the page's information.
+1. **Semantic icons**, which provide information that isn't present on the page. For example, these can represent status, indicate an alert message type, or replace text as button labels. The meaning of a semantic icon must be provided in alternative ways to the user. Additionally, you should follow these guidelines:
+    - Add a label for the icon in tooltip text that displays on hover and focus.
+    - For interactive elements (like `<a>` and `<button>`), where an icon is used as the label instead of text, set a label as an `aria-label`. 
 
-- Add a label for the icon in tooltip text that displays on hover and on focus for focusable elements. 
+      Example: 
 
-- For **interactive elements** like `<a>` and `<button>` where an icon is used as the label instead of text, provide the label on the interactive element using aria-label. 
+        ```html noLive
+        <button class="..." aria-label="Close Dialog">
+          <i class="..." aria-hidden="true"></i>
+        </button>
+        ```
 
-  Example: 
+#### **Non-interactive icons**
 
-    ```html noLive
-    <button class="..." aria-label="Close Dialog">
-      <i class="..." aria-hidden="true"></i>
-    </button>
-    ```
+For **non-interactive icons**, include `.pf-screen-reader` text near the icon. Depending on the component, this text might not be a direct sibling to the icon element.  
 
-- For **non-interactive icons**, include .pf-screen-reader text near the icon. Depending on the component, the .pf-screen-reader text might not be a direct sibling to the icon element.  
+Example: 
 
-  Example: In the alert component, the icon label text is adjacent to the message. This way, when role="alert" is added to .pf-v5-c-alert__body for dynamically displayed alerts, the type of message is announced along with the message text.
+In the alert component, the icon label text is adjacent to the message. This way, when `role="alert"` is added to `.pf-v5-c-alert__body` for dynamically displayed alerts, the type of message is announced along with the message text.
```

Comment:
Would the whole example be removed, then? I removed it in my last push to visualize, but can add it back and modify.

Reply:
I think keeping the example could be useful, at least to show how using hidden text can be used.

Reply:
We can definitely update this example to be more current. A lot of this content has been around for years, and I know that a few years back I did a good bit of work to try to get alerts to work across screen readers so I'm sure that things have changed since this content was originally created. So no objection to removing that part of the example. :)